### PR TITLE
Fixed CreateRelatedTermHandlerTest, GetTransactLinByStreetcodeIdHandl…

### PR DESCRIPTION
**Pull request by 666HellPlay666**
…erTest, and created GetAllMainTeamHandlerTest

dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/RelatedTerm/Create/CreateRelatedTermHandlerTests.cs [hunk 1]
```diff
@@ -56,70 +59,69 @@ public async Task Handle_Should_Return_Error_When_Related_Term_Is_Null()
             var result = await handler.Handle(command, CancellationToken.None);
 
             // Assert
-            Assert.Multiple(
-           () => Assert.True(result.IsFailed),
-           () => Assert.Equal("Cannot create new related word for a term!", result.Errors.First().Message)
-           );
+            result.IsFailed.Should().BeTrue();
```
Comment (DanyilTerentiev) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/RelatedTerm/Create/CreateRelatedTermHandlerTests.cs:62]: Why don't you use Assert there?

File: Streetcode/Streetcode.XUnitTest/MediatRTests/Team/GetAllMainTeamHandlerTest.cs [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            _mockMapper = new Mock<IMapper>();
+            _mockRepo = new Mock<IRepositoryWrapper>();
+        }
+
+        [Fact]
+        public async Task Handle_ReturnsSuccess()
+        {
+            // Arrange
+
+            _mockRepo.Setup(r => r.TeamRepository.GetAllAsync(null,
```
Comment (DanyilTerentiev) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/Team/GetAllMainTeamHandlerTest.cs:29]: Make a function for setups, please

File: Streetcode/Streetcode.XUnitTest/MediatRTests/Team/GetAllMainTeamHandlerTest.cs [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+                Positions = new List<Positions>(),
+                TeamMemberLinks = new List<TeamMemberLink>()
+            },
+            new TeamMember
+            {
+                Id=2,
+                IsMain = true,
+                Positions = new List<Positions>(),
+                TeamMemberLinks = new List<TeamMemberLink>()
+            }
```
Comment (Tysyatsky) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/Team/GetAllMainTeamHandlerTest.cs:83]: check the code style here.

**Review by DanyilTerentiev — APPROVED** [review]
Approved

**Review by MaksBrat — APPROVED** [review]

File: Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/RelatedTerm/Create/CreateRelatedTermHandlerTests.cs [hunk 4]
```diff
@@ -56,70 +59,69 @@ public async Task Handle_Should_Return_Error_When_Related_Term_Is_Null()
...
             _mapperMock.Setup(m => m.Map<Entity>(It.IsAny<RelatedTermDTO>())).Returns(entity);
-            _repositoryWrapperMock.Setup(r => r.RelatedTermRepository.
-                GetAllAsync(It.IsAny<Expression<Func<Entity, bool>>>(),
-                It.IsAny<Func<IQueryable<Entity>, IIncludableQueryable<Entity, object>>>()))
-            .ReturnsAsync(existingTerms);
+            _repositoryWrapperMock.Setup(r => r.RelatedTermRepository
+                .GetAllAsync(It.IsAny<Expression<Func<Entity, bool>>>(),
+                    It.IsAny<Func<IQueryable<Entity>, IIncludableQueryable<Entity, object>>>()))
+                .ReturnsAsync(existingTerms);
+
```
Comment (NadiaKishchuk) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/RelatedTerm/Create/CreateRelatedTermHandlerTests.cs:80]: please, use a separate method for setup

File: Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/RelatedTerm/Create/CreateRelatedTermHandlerTests.cs [hunk 5]
```diff
@@ -56,70 +59,69 @@ public async Task Handle_Should_Return_Error_When_Related_Term_Is_Null()
...
-            Assert.Multiple(
-             () => Assert.True(result.IsFailed),
-             () => Assert.Equal("Слово з цим визначенням уже існує", result.Errors.First().Message),
-             () => _repositoryWrapperMock.Verify(r => r.RelatedTermRepository.Create(entity), Times.Never),
-             () => _repositoryWrapperMock.Verify(r => r.SaveChangesAsync(), Times.Never)
-             );
+            Assert.True(result.IsFailed);
+            Assert.Equal("Слово з цим визначенням уже існує", result.Errors.First().Message);
+            _repositoryWrapperMock.Verify(r => r.RelatedTermRepository.Create(entity), Times.Never);
+            _repositoryWrapperMock.Verify(r => r.SaveChangesAsync(), Times.Never);
```
Comment (NadiaKishchuk) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/RelatedTerm/Create/CreateRelatedTermHandlerTests.cs:91]: please, don't use magic strings

File: Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/RelatedTerm/Create/CreateRelatedTermHandlerTests.cs [hunk 6]
```diff
@@ -56,70 +59,69 @@ public async Task Handle_Should_Return_Error_When_Related_Term_Is_Null()
...
 
             // Assert
-            Assert.Multiple(
-             () => Assert.True(result.IsFailed),
-             () => Assert.Equal("Cannot save changes in the database after related word creation!", result.Errors.First().Message),
-             () => repositoryMock.Verify(r => r.RelatedTermRepository.Create(entity), Times.Once),
-             () => repositoryMock.Verify(r => r.SaveChangesAsync(), Times.Once)
-             );
+            Assert.True(result.IsFailed);
+            Assert.Equal("Cannot save changes in the database after related word creation!", result.Errors.First().Message);
```
Comment (NadiaKishchuk) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/RelatedTerm/Create/CreateRelatedTermHandlerTests.cs:122]: the same here

**Review by NadiaKishchuk — CHANGES_REQUESTED** [review]

**Review by NadiaKishchuk — APPROVED** [review]

File: Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/RelatedTerm/Create/CreateRelatedTermHandlerTests.cs [hunk 7]
```diff
@@ -48,78 +83,70 @@ public async Task ShouldReturnSuccessfully_WhenRelatedTermAdded(int termId, stri
...
             var result = await handler.Handle(command, CancellationToken.None);
 
             // Assert
-            Assert.Multiple(
-           () => Assert.True(result.IsFailed),
-           () => Assert.Equal("Cannot create new related word for a term!", result.Errors.First().Message)
-           );
+            Assert.True(result.IsFailed);
+            Assert.False(result.IsSuccess);
+            Assert.Empty(result.Errors);
```
Comment (Tysyatsky) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/RelatedTerm/Create/CreateRelatedTermHandlerTests.cs:96]: why do you removed multiple?

File: Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/RelatedTerm/Create/CreateRelatedTermHandlerTests.cs [hunk 8]
```diff
@@ -48,78 +83,70 @@ public async Task ShouldReturnSuccessfully_WhenRelatedTermAdded(int termId, stri
...
-            Assert.Multiple(
-             () => Assert.True(result.IsFailed),
-             () => Assert.Equal("Слово з цим визначенням уже існує", result.Errors.First().Message),
-             () => _repositoryWrapperMock.Verify(r => r.RelatedTermRepository.Create(entity), Times.Never),
-             () => _repositoryWrapperMock.Verify(r => r.SaveChangesAsync(), Times.Never)
-             );
+            Assert.True(result.IsFailed);
+            Assert.False(result.IsSuccess);
+            Assert.Empty(result.Errors);
+            VerifyCreateAndSaveChangesNever();
```
Comment (Tysyatsky) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/RelatedTerm/Create/CreateRelatedTermHandlerTests.cs:120]: same here

File: Streetcode/Streetcode.XUnitTest/MediatRTests/Team/GetAllMainTeamHandlerTest.cs [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+                It.IsAny<IEnumerable<TeamMember>>()))
+                .Returns(GetListTeamDTO());
+        }
+
+        public GetAllMainTeamHandlerTests()
+        {
+            _mockMapper = new Mock<IMapper>();
+            _mockRepo = new Mock<IRepositoryWrapper>();
+            SetupMocks();
+
```
Comment (Tysyatsky) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/Team/GetAllMainTeamHandlerTest.cs:35]: spacing

**Pull request merged**